src/meta-srv/src/selector/round_robin.rs

```diff
@@ -125,6 +127,35 @@ impl Selector for RoundRobinSelector {
     }
 }
 
+pub struct FlowPeerAllocator {
```

Comment:
Could we move the `FlowPeerAllocator` elsewhere? I believe only xxx_selector should stay here.

Reply:
We can create a similar file: https://github.com/GreptimeTeam/greptimedb/blob/a58256d4d3f031470dd765a8cbc849dbbba6863a/src/meta-srv/src/table_meta_alloc.rs